Godeps/_workspace/src/github.com/endophage/gotuf/store/httpstore.go

```diff
@@ -113,8 +117,17 @@ func (s HTTPStore) SetMeta(name string, blob []byte) error {
 	if err != nil {
 		return err
 	}
-	_, err = s.roundTrip.RoundTrip(req)
-	return err
+	resp, err := s.roundTrip.RoundTrip(req)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode == http.StatusNotFound {
+		return ErrMetaNotFound{}
+	} else if resp.StatusCode != http.StatusOK {
```

Comment:
Is the spec explicit about 200 being the only acceptable success code for these POST operations?